### DISCUSSION TITLE
[venus]全组件metrics指标监控

### DISCUSSION THREAD

**Discussion by hunjixin** [filecoin-project/venus#4950] [Ideas]
### Checklist

- [X] This is **not** a new feature or an enhancement to the Filecoin protocol. If it is,  please open an [FIP issue](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/venus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the venus forum](https://github.com/filecoin-project/venus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated improvement to propose.

### Venus component

- [X] venus daemon - [chain service] chain sync
- [X] venus auth - [chain service] authentication
- [X] venus messager - [chain service] message management (mpool)
- [X] venus gateway - [chain service] gateway
- [X] venus miner - [chain service] mining and block production
- [X] venus sealer/worker - sealing
- [X] venus sealer - proving (WindowPoSt)
- [X] venus market - storage deal
- [X] venus market - retrieval deal
- [X] venus market - data transfer
- [ ] venus light-weight client
- [ ] venus JSON-RPC API
- [ ] Other

### Improvement Suggestion

venus组件重要指标监控支持，方便运维和用户直接方便进行指标监控，发生异常是能够及时干预修复。

**Comment by Fatman13** [comment]
这几个怎么样？ 

- Block validation time
- Memory / CPU usage
- Number of goroutines
- IPLD block read latency
- Bandwidth usage
